tests/test_cpp/polars/TestWindowProcessor.cpp

```diff
@@ -14,6 +14,60 @@
 namespace WindowProcessorTests {
 using Series = polars::Series;
 
+// Tests for rolling median
```

Comment:
These are tests that I have added that should pass once we make everything work. I will also move this to a separate PR.

Reply:
Add `DISABLED_` like above